### DISCUSSION TITLE
My Artisan 2.4.6, the font is not self-adjusting

### DISCUSSION THREAD

**Discussion by Diomioleo** [artisan-roaster-scope/artisan#717] [Q&A] [answered]
Hi Would like to check how can I set so that my Artisan font can be self adjusting, or where I can set the font smaller as well as the control buttons?

![image](https://user-images.githubusercontent.com/93377102/139615732-f188e9c5-bc90-4a17-b7f2-1f127079363d.png)

**Comment by poundy** [comment, marked as answer]
Config --> Curves --> UI tab has the spot to change your UI font. You can also adjust via a theme at Config --> Themes.

Having said that, I suspect you have a very specific setup so you should provide a full copy of your Artisan Configuration so we can see what you have previously set. You could also tell us about what you're running this on and what resolution display, as it looks "unique" ;-)

**Reply by Diomioleo** [reply]
Poundy，thank you for the help，yes you are right，I got this config from my roaster vendor，I will go for the UI tab，thanks

**Comment by fcebreros** [comment]
You can adjust the font size in:
Config/Curves/UI
Graph

Greetings
FC
